server/handler/query.go

```diff
@@ -103,40 +104,53 @@ func Query(db service.SQLDB) RequestProcessFunc {
 			return nil, dbError(err)
 		}
 
-		defer rows.Close()
+		return resp, nil
+	}
+}
 
-		columnNames, columnTypes, err := columnsInfo(rows)
-		if err != nil {
-			return nil, err
-		}
+func queryContext(ctx context.Context, conn *sql.Conn, queryRequest queryRequest) (*serializer.Response, error) {
```

Comment:
is it really neede to shadow `queryRequest` type?

Reply:
sorry, I don't follow what you mean here. Can you elaborate?

Reply:
the name of the parameter is `queryRequest` and the type of this parameter is `queryRequest`.
because of it you don't have access to the type anymore inside the function and it is a little bit confusing that `queryRequest` outside of this function is a type but inside an instance of this type.

Reply:
you are right, I just copied the code we had before and didn't realize that. Thanks.
I've renamed the var, and because it's a small change I already squashed it to make the branch ready to merge.